app/form_builders/gds_adp_form_builder.rb

```diff
@@ -0,0 +1,7 @@
+class GdsAdpFormBuilder < AdpFormBuilder
```

Comment:
As ADP is an old name for CCCD can it be dropped from this name, and so become `GdsFormBuilder`?

I presume that the ultimate goal is that this form builder will completely replace `AdpFormBuilder`.

Reply:
This new form builder is to show the inheritance & composition of AdpFormBuilder and GOVUKDesignSystemFormBuilder.

This should be short lived and will be removed once we have migrate to and using the GOVUKDesignSystemFormBuilder as the default.